src/io/image_olm_parser.rs

```diff
@@ -26,7 +28,7 @@ type PixelKeys = BiMap<usize, Rgb<u8>>;
 
 pub fn render(
     filename: &str,
-    graph: Graph,
+    graphs: Vec<Graph>,
```

Comment:
Continuing the comments on not returning a vec of graphs. This function should just take a vec of vertices.